README.md

```diff
@@ -425,24 +400,18 @@ mq.index("my-first-multimodal-index").add_documents(
     [
         {
             "Title": "Flying Plane",
-            "captioned_image": {
-                "caption": "An image of a passenger plane flying in front of the moon.",
-                "image": "https://raw.githubusercontent.com/marqo-ai/marqo/mainline/examples/ImageSearchGuide/data/image2.jpg",
-            },
+            "caption": "An image of a passenger plane flying in front of the moon.",
+            "image": "https://raw.githubusercontent.com/marqo-ai/marqo/mainline/examples/ImageSearchGuide/data/image2.jpg",
         },
         {
             "Title": "Red Bus",
-            "captioned_image": {
-                "caption": "A red double decker London bus traveling to Aldwych",
-                "image": "https://raw.githubusercontent.com/marqo-ai/marqo/mainline/examples/ImageSearchGuide/data/image4.jpg",
-            },
+            "caption": "A red double decker London bus traveling to Aldwych",
+            "image": "https://raw.githubusercontent.com/marqo-ai/marqo/mainline/examples/ImageSearchGuide/data/image4.jpg",
         },
         {
             "Title": "Horse Jumping",
-            "captioned_image": {
-                "caption": "A person riding a horse over a jump in a competition.",
-                "image": "https://raw.githubusercontent.com/marqo-ai/marqo/mainline/examples/ImageSearchGuide/data/image1.jpg",
-            },
+            "caption": "A person riding a horse over a jump in a competition.",
+            "image": "https://raw.githubusercontent.com/marqo-ai/marqo/mainline/examples/ImageSearchGuide/data/image1.jpg",
         },
     ],
     # Create the mappings, here we define our captioned_image mapping 
```

Comment:
have we tested this example?

Reply:
The query is working and returning response. However, since we have only one tensor filed, I think the tensor field needs to be `image`  field to make more sense, right?

Reply:
Have checked this with Farshid, and he confirmed using `tensor_fields=["captioned_image"]` as tensor field.